docs/developer-guide/nodejs-api.md

```diff
@@ -427,7 +437,16 @@ The return value is an object containing the results of the linting operation. H
                 line: 1,
                 column: 13,
                 nodeType: "ExpressionStatement",
-                fix: { range: [12, 12], text: ";" }
+                fix: { range: [12, 12], text: ";" },
+                suggestions: [
+                    {
+                        desc: 'Insert a semicolon',
```

Comment:
Should a more realistic example of a suggestion be used here? The `no-useless-escape` suggestions seems like a decent candidate for understandable realistic suggestions, but those don't correspond to the `semi` rule and I'm not sure if there would ever be a case for a rule that would have both an autofix capability and a suggestion.